src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -70,15 +104,33 @@ inline void image(GL::Texture2D& texture, const Vector2& size,
     padding
 @param backgroundColor  Background color, default @cpp 0x00000000_rgbaf @ce
 @param tintColor        Tint color, default @cpp 0xffffffff_rgbaf @ce
-@m_since_{integration,2019,10}
+@m_deprecated_since_latest Use @ref imageButton(const char*, GL::Texture2D&, const Vector2&, const Range2D&, const Color4&, const Color4&) instead.
```

Comment:
I'd prefix this with `Using an implicit ID is no longer possible in newer ImGui.` or something similar, so it's clear why this got deprecated without having to look at the source.